tests/unit/test_charm.py

```diff
@@ -256,7 +256,7 @@ def test_given_vault_snap_uninstalled_when_configure_then_vault_snap_installed(
 
         self.mock_snap_cache.assert_called_with()
         vault_snap.ensure.assert_called_with(
-            SnapState.Latest, channel="1.15/stable", revision="2226"
+            SnapState.Latest, channel="1.16/stable", revision="2226"
```

Comment:
Do we have the same revision number on both channels?

Reply:
We don't. Fixed